--- a/net/minecraft/client/renderer/texture/TextureAtlasSprite.java
+++ b/net/minecraft/client/renderer/texture/TextureAtlasSprite.java
@@ -238,7 +238,7 @@
       } else if (this.field_228963_g_ != null) {
          if (!RenderSystem.isOnRenderThread()) {
             RenderSystem.recordRenderCall(() -> {
-               p_228967_0_.func_228993_a_();
+               this.field_228963_g_.func_228993_a_();
             });
          } else {
             this.field_228963_g_.func_228993_a_();
